lib/optimizely.rb

```diff
@@ -523,6 +524,47 @@ def close
       @event_processor.stop! if @event_processor.respond_to?(:stop!)
     end
 
+    def get_optimizely_config # rubocop:disable Naming/AccessorMethodName
+      # Get OptimizelyConfig object containing experiments and features data
+      # Returns Object
+      #
+      # OptimizelyConfig Object Schema
+      # {
+      #   'experimentsMap' => {
+      #     '111111' => {
+      #       'id' => '111111',
+      #       'key' => 'my-fist-experiment'
+      #       'variationsMap' => {
+      #         '121212' => {
+      #           'id' => '121212',
+      #           'key' => 'variation_1',
+      #           'variablesMap' => {
+      #             '222222' => {
+      #               'id' => '222222',
+      #               'key' => 'age',
+      #               'type' => 'integer',
+      #               'value' => '0',
+      #             }
+      #           }
+      #         }
+      #       }
+      #     }
+      #   },
+      #   'featuresMap' => {
+      #     '333333' => {
+      #       'id' => '333333',
+      #       'key' => 'awesome-feature',
+      #       'experimentsMap' => Object,
+      #       'variablesMap' => Object,
+      #     }
+      #   },
+      #   'revision' => '13',
+      # }
+      #
+      optimizely_config = OptimizelyConfig.new(project_config)
```

Comment:
Add "unless is_valid" guard like other API calls, returning null when not ready.

Reply:
Done